charts/presto/templates/deployment-coordinator.yaml

```
@@ -37,6 +37,9 @@ spec:
       annotations:
         checksum/catalog: {{ include (print $.Template.BasePath "/configmap-catalog.yaml") . | sha256sum }}
         checksum/coordinator: {{ include (print $.Template.BasePath "/configmap-coordinator.yaml") . | sha256sum }}
+        {{- with .Values.coordinator.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
```

```suggestion
        {{- with .Values.coordinator.podAnnotations }}
          {{- toYaml . | nindent 8 }}
        {{- end }}
```
Could you please align the indentation to follow the common style (similar to [service labels](https://github.com/prestodb/presto-helm-charts/blob/5aa7ad008578fec10915515bf76a6d868129a322/charts/presto/templates/service.yaml#L6-L13))?

---

done.